### PR TITLE
No conda's gcc

### PR DESCRIPTION
**Pull request by ocefpaf**
Also re-rendered.

### CODE REVIEW

File: ci_support/run_docker_build.sh [hunk 1]
```diff
@@ -37,16 +39,11 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
+conda update --yes --all
+#conda install --yes conda-build==1.18.2
```
Comment (pelson) [inline ci_support/run_docker_build.sh]: ?

Comment (ocefpaf) [inline ci_support/run_docker_build.sh]: This one needs to unpack `.z` and that was included `conda-build >1.18.2`

Comment (pelson) [inline ci_support/run_docker_build.sh]: I see. Slightly worrying that some feedstocks *require* one version of conda-build and other feedstocks *require* another version.

Anyway, :+1:

Comment (ocefpaf) [inline ci_support/run_docker_build.sh]: Ideally we would always use latest, but we are being held back due to a bug. (I can't remember which one!)

For this package I need this https://github.com/conda/conda-build/pull/731

**Pull request merged**